Add initial documentation for Ambiences.

MPolleke opened the pull request:
Hello Sailors,
This is a pull request to add some initial documentation for Ambiences.

The documentation for Ambiences is scattered around, sparse or non-existing. This is meant as an effort to build up and concentrate that documentation.
It feels a bit like stabbing in the dark :)

If you need any changes before merging, please let me know. I can imagine the place in the tree of documentation is up for debate, as it's not an app, and it's not the platform.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/sailfishos/docs.sailfishos.org?pullRequest=333) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/sailfishos/docs.sailfishos.org?pullRequest=333) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/sailfishos/docs.sailfishos.org?pullRequest=333) it.</sub>

vigejolla commented on Develop/Ambience/README.md:
```diff
@@ -0,0 +1,87 @@
+title: Ambience
+permalink: Develop/Ambience/
+layout: default
+nav_order: 700
```
Like you wrote, the place in document tree is up to debate. I imagine that we might want to move it to some other place later. Anyway, I would put it right under L10n in the, as the documents after that are more generic instructions. So, maybe `nav_order: 450` ?

vigejolla commented on Develop/Ambience/README.md:
```diff
@@ -0,0 +1,94 @@
+nav_order: 700
+---
+
+## This article requires verification and updating.
```
Please remove this line - this is what we are doing now and such comments should be removed before merging

vigejolla commented on Develop/Ambience/README.md:
```diff
@@ -0,0 +1,94 @@
+
+## What is an Ambience?
+
+An Ambience defines the colors and other configuration used on your phone. Silica and all native apps will use the configuration from the Ambience.
```
I would add a link to `/Support/Help_Articles/Ambiences/` here, as the ambience concept is described in more detail there (from user point of view). And then we could add a link back here from there - you can do that already in this PR if you want.

vigejolla commented on Develop/Ambience/README.md:
```diff
@@ -0,0 +1,94 @@
+
+An Ambience defines the colors and other configuration used on your phone. Silica and all native apps will use the configuration from the Ambience.
+
+The system service `ambienced` will set the Ambience upon boot, or switch to a different Ambience upon request. The license of Ambienced is proprietary.
```
I wonder if we could still clarify this a little bit. Ambienced runs as a *user* service.

vigejolla commented on Develop/Ambience/README.md:
```diff
@@ -0,0 +1,94 @@
+
+## Content of an .ambience file
+
+The .ambience file is located under `/usr/share/ambience/ambience-%name/` where %name is the name of the ambience, for example "ambience-my-special-ambience/".
```
I don't think they have to start with "ambience-" in the name. At least the default ones don't have that.

vigejolla commented on Develop/Ambience/README.md:
````diff
@@ -0,0 +1,94 @@
+Set image as the ambience:
+
+```nosh
+dbus-send --session --print-reply --dest=com.jolla.ambienced /com/jolla/ambienced com.jolla.ambienced.setAmbience string:"file://home/<user>/Pictures/imag    e.jpg"
````
Extra spaces in the filename

vigejolla commented on Develop/Ambience/README.md:
```diff
@@ -0,0 +1,94 @@
+
+In the settings for the Sailfish Browser, you can select if the browser should use a light theme, dark theme or follow the Ambience.
+The dark mode will only work if the website supports a dark theme.
+It might be possible to create an Ambience that will make the browser follow the theme for dark mode.
```
It's definitely possible; some of the bundled ambiences use dark mode and some use light mode.

vigejolla reviewed (changes requested):
Great work! It needs some adjustments before merging though.

MPolleke commented:
Thank you very much for the feedback :)

I addressed the requested changes. I added a few more changes and updates that I saw fit.

I haven't managed to make the browser use dark mode, not with the other ambiences aither, like airy and glacial. And yes, this started with a support question :) I hope someone manages to clarify how to do that.

vigejolla commented on Develop/Ambience/README.md:
```diff
@@ -0,0 +1,109 @@
+
+## What is an Ambience?
+
+Read more about [Ambiences](Support/Help_Articles/Ambiences/) and how to use them.
```
```suggestion
Read more about [Ambiences](/Support/Help_Articles/Ambiences/) and how to use them.
```

vigejolla commented on Develop/Ambience/README.md:
````diff
@@ -0,0 +1,109 @@
+/home/defaultuser/.local/share/system/privileged/Ambienced/ambienced.sqlite
+```
+
+Settings of the current active Ambience are located in the dconf database at "/desktop/jolla/theme". Dconf is a settings database per user located in the file ".config/dconf/user". These settings can be shown bu running the shell command:
````
I would drop the sentence 'Dconf is a settings database per user located in the file ".config/dconf/user"' as that is dconf implementation detail and not really related to ambiences.

vigejolla reviewed (changes requested):
Almost there :)

vigejolla commented:
> I haven't managed to make the browser use dark mode, not with the other ambiences aither, like airy and glacial

What do you mean? Just select an ambience with "darkonlight" color scheme and the browser will use light mode, and ambience with "lightondark" will cause it to use dark mode. Works on e.g. github.com.

vigejolla commented:
> And I must be confused, I tested on duckduckgo.com, which works on my desktop but not my phone.

duckduckgo.com works on my phone.

There's still that one '/' missing from the link in line 10. I suppose I can commit it as well - if you are satisfied with this I'm ready to merge this.

MPolleke commented:
Thank you, I had missed that.

And yes, I would be happy if you can merge this. I will try to add some updates, I found some more scattered docs :)

vigejolla reviewed (approved):
LGTM

vigejolla commented:
Thank you!
